packages/status-react/src/styles/config.tsx

```diff
@@ -161,6 +161,182 @@ export const darkTheme = createTheme({
   },
 })
 
+// todo: check source licenses
+/**
+ * Base styles.
+ *
+ * @see https://tailwindcss.com/docs/preflight for styles source
+ * @see https://unpkg.com/tailwindcss@3.1.8/src/css/preflight.css for styles source
+ * @see https://github.com/codesandbox/sandpack/blob/main/sandpack-react/src/contexts/themeContext.tsx for styles source
```

Comment:
This link has been [moved](https://github.com/codesandbox/sandpack/pull/606/files#diff-ef320de67c942aac6167958e5b1f652adb01179be4aba48f40651de9cea7c50c).